platform/core/src/services/DisplaySetService/extractFrameMetadata.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Extract frame metadata from a multiframe metadata. All information related to
+ * multiframe is removed from the instance metadata
+ */
+const extractFrameMetadata = (frame, instance) => {
```

Comment:
I thought this code already existed somewhere?  If it does, we should have a single copy.  If you are just moving it, please document that - at least in a comment.

Reply:
This was a temporary file erased in this commit